src/server/routes/chat.route.js

```diff
@@ -0,0 +1,15 @@
+import express from 'express';
+import * as pushUtil from '../utils/push.util';
+
+const router = express.Router();
+
+router.post('/callback', (req, res, next)=> {
```

Comment:
Does only one method is needed in chat.route.js?
Then just making url as ```'/'``` would be one of the way we can do.